contracts/VM.sol

```diff
@@ -41,43 +43,57 @@ abstract contract VM {
         bytes memory outdata;
 
         uint256 commandsLength = commands.length;
-        for (uint256 i; i < commandsLength; i=_uncheckedIncrement(i)) {
+        for (uint256 i; i < commandsLength; i = _uncheckedIncrement(i)) {
             command = commands[i];
             flags = uint256(uint8(bytes1(command << 32)));
 
             if (flags & FLAG_EXTENDED_COMMAND != 0) {
-                indices = commands[i++];
```

Comment:
I believe your change here will introduce an off-by-one bug

Reply:
I see, this is addressing the issue covered in the audit. This change is correct.